test/test_quantized.py

```diff
@@ -485,9 +485,47 @@ def test_max_pool2d_nhwc(self, X, kernel, stride, dilation, padding):
             qa, kernel_size=_pair(kernel),
             stride=_pair(kernel if stride is None else stride),
             padding=_pair(padding), dilation=_pair(dilation))
+        print(a_hat.dequantize())
+        print(a_ref)
         self.assertEqual(a_ref, a_hat.dequantize(),
                          message="ops.quantized.max_pool2d results are off")
 
+    @given(X=hu.tensor(shapes=hu.array_shapes(min_dims=3, max_dims=4,
```

Comment:
Please add a test for NHWC layout, similar to what we have for maxpool https://github.com/pytorch/pytorch/pull/25899/files#diff-d7c934f715ae077dce41cdfa38bf0fd1L442